### PR TITLE
[NEWSTACK-474] update CSI sidecar versions and permission for support k8s 1.22

### PR DESCRIPTION
**Pull request by dsupure**
**Summary of changes**
* [NEWSTACK-474] update CSI sidecar versions and permission for support k8s 1.22

### CODE REVIEW

File: pure-pso/values.yaml [hunk 1]
```diff
@@ -80,27 +80,27 @@ flashblade:
   exportRules: "*(rw,no_root_squash)"
 images:
   plugin:
-    name: purestorage/k8s
-    tag: v6.2.2
+    name: pc2-dtr.dev.purestorage.com/purestorage/k8s
```
Comment (pure-jliao) [inline pure-pso/values.yaml:83]: why is this using pc2-dtr?

**Review by pure-jliao — COMMENTED** [review]
we use to bump cockroach operator version together, but if there's no change think it's fine.

**Review by david-enli — APPROVED** [review]
LGTM, once we change the repo from pc2-dtr back to k8s

File: pure-pso/values.yaml (hunk 1 above)
Comment (smaanika) [reply pure-pso/values.yaml]: not sure how it got missed, but it has been fixed now.

**Comment by dsupure** [comment]
Thanks team!

**Pull request merged**